regexps.json

```diff
@@ -0,0 +1,5 @@
+{
+    "re1": "([0-9]+) (day?|week?|month?|year?)(:?s)? ago",
+    "re2": "(?:last|previous) (Mon|Tues|Wednes|Thurs|Fri|Satur|Sun)(?:day)?",
+    "re3": "([0-9]+)(?:th)? (Jan(?:uary)?|Feb(?:ruary)?|March|April|May|June|July|August|Sept(?:ember)?|Oct(?:ober)?|Nov(?:ember)?|Dec(?:ember)?)"
```

Comment:
Same here. I think everything is there except `mar, Apr, Jun, Jul, aug`. That's about all. (I think a change here will also cause changes in the lists above.)